build.yaml

```diff
@@ -102,7 +102,24 @@ release:
                   find build -type f -name "*.${PACKAGE_TYPE}" -exec mv {} packages \; 
                   rm -rf build
                   PHP_TOKENS=($(echo ${PHP_VERSION} | tr "-" " "))
-                  sudo ${PACKAGE_UTILITY_REMOVE} ${PHP_TOKENS[0]}-common
+                  # Installing packaging and testing connection
+                  sudo ${PACKAGE_INSTALL} packages/${PHP_TOKENS[0]}*.${PACKAGE_TYPE}
+                  php <<EOF
+                  <?php
+                  \$cluster = Cassandra::cluster()->build();
+                  \$keyspace = 'system';
+                  try{
+                    \$session = \$cluster->connect(\$keyspace);
```

Comment:
The keyspace is optional. It's not needed.

(Ref: https://docs.datastax.com/en/developer/php-driver/1.3/api/Cassandra/interface.Cluster/#method-connect)